### DISCUSSION TITLE
Open SSL error

### DISCUSSION THREAD

**Discussion by hschoigp** [iiasa/message_ix#699] [Getting started]
I encountered an error while installing message ix in my PC as below.
It says that OpenSSL is not available in my PC. Please, help me with tacking this problem.
I searched on the internet and other tips doesn't solve this issue. 

![캡처](https://user-images.githubusercontent.com/96801257/227482706-fdbdf420-e173-49aa-b4db-2e323aa0d59c.JPG)

**Comment by khaeru** [comment]
This appears to be an issue with Anaconda, rather than anything to do with message_ix, so this isn't really the right place to ask the question.

Regardless:

> I searched on the Internet and other tips doesn't solve this issue.

For me, a search of "CondaSSLError: OpenSSL appears to be unavailable on this machine" leads to https://github.com/conda/conda/issues/11795#issuecomment-1335666474 as the first result. Is this what you meant by "other tips", or something else? Did you see this, read it and try it (or subsequent comments in the same thread)?
